eclair-core/src/test/scala/fr/acinq/eclair/router/RouterSpec.scala

```diff
@@ -536,84 +536,6 @@ class RouterSpec extends BaseRouterSpec {
     assert(route2.finalHop_opt.contains(trampolineHop))
   }
 
-  test("routes found (with blinded hops)") { fixture =>
```

Comment:
I'm not sure we should remove these tests entirely, they are useful still to test that the `ignore` and `pending` mechanisms work correctly. But it's true that the hack I added makes it fail. What about:

- updating `BlindedRecipient.fromPaths` to explicitly take a `duplicatePaths: Int` argument and use that instead of always duplicating 3 times
- in tests, we can set it to `1` to remove the duplication and test the router behavior

It will be easier to update when we re-work that part to use a cleaner model.

Reply:
Done